patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -9,18 +9,19 @@
  {
      private static final Logger field_184243_a = LogManager.getLogger();
      private static final List<ItemStack> field_190535_b = Collections.<ItemStack>emptyList();
-@@ -177,6 +177,10 @@
+@@ -177,6 +177,11 @@
      private boolean field_184237_aG;
      private final double[] field_191505_aI;
      private long field_191506_aJ;
 +    /**
 +     * Setting this to true will prevent the world from calling {@link #onUpdate()} for this entity.
 +     */
 +    public boolean updateBlocked;
++    public boolean isWorldActiveInChunk = false;
```

Comment:
This should just be named isInWorld; it has no relation to a chunk. An Entity can exist in a World but not exist in any Chunk (I don't like that this is possible, but sadly Mojang did this)